KeyVault.Acmebot/Providers/GoDaddyProvider.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using KeyVault.Acmebot.Options;
+
+using Newtonsoft.Json;
+
+namespace KeyVault.Acmebot.Providers
+{
+    public class GoDaddyProvider : IDnsProvider
+    {
+        public GoDaddyProvider(GoDaddyOptions options)
+        {
+            _client = new GoDaddyClient(options.ApiKey, options.ApiSecret, "https://api.godaddy.com");
+        }
+
+        private readonly GoDaddyClient _client;
+
+        public int PropagationSeconds => 10;
```

Comment:
Isn't 10 seconds too short, has GoDaddy confirmed that it will reflect quickly?

Reply:
Lacking better information, I have made it 60 seconds as that is what other providers have used.